lib/list.gd

```diff
@@ -2376,36 +2376,22 @@ DeclareGlobalFunction( "ListWithIdenticalEntries" );
 ##
 #F  PlainListCopy( <list> ) . . . . . . . .  make a plain list copy of a list
 ##
+##  <#GAPDoc Label="PlainListCopy">
 ##  <ManSection>
 ##  <Func Name="PlainListCopy" Arg='list'/>
 ##
 ##  <Description>
+##  This operation returns a list equal to its argument, in a plain list
+##  representation.
 ##  This is intended for use in certain rare situations,
-##  such as before objectifying.
+##  such as before objectifying or calling some kernel functions.
```

Comment:
Just for clarification, to rule out the interpretation "objectifying some kernel functions or calling some kernel functions"
```suggestion
##  such as before objectifying, or calling some kernel functions.
```